### PR TITLE
Wait for 848, then merge this: 767 - Create Werk Modal & User Permissions, Connect Wallet

### PR DESCRIPTION
**Pull request by nagmak**
### What changes were made?

- Added a werk modal for posting a job on Marketplace
- Added User Permissions to marketplace, and SOW builder, and changed the Connect wallet flow to include a logged out message

### Is there any background info about the change?

- Will ensure users can log in if they refresh the page or disconnect from the site on any page that uses this flow

### Are there any state changes? Mention key ones (if any)

A few refactored ones

### Screenshots or Gifs✨
Screen Shot 2022-05-26 at 2 40 24 PM
Screen Shot 2022-05-26 at 2 39 43 PM
Screen Shot 2022-05-26 at 2 39 58 PM

### Any new dependencies? Why were they added?

NA

### Shortcut Link

https://app.shortcut.com/twali/story/853/user-permissions-connect-wallet-on-sow-builder-flow
https://app.shortcut.com/twali/story/767/sow-create-werk-modal

### CODE REVIEW

File: components/Marketplace/SortBounty.tsx [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
+import SelectWerkModal from './SelectWerkModal/SelectWerkModal'
+
 export const SortBounty = ({ contracts, onChange }) => {
+
+  const {
+    isOpen: isSelectWerkModalOpen,
+    onOpen: onSelectWerkModalOpen,
+    onClose: onSelectWerkModalClose,
+  } = useDisclosure();
+
```
Comment (NickGonzalez04) [inline components/Marketplace/SortBounty.tsx:13]: Has merge conflict here with the options below on line 15 and 16. I think the branch just needs to be updated with the recent changes for merged PRs into staging.

**Review by NickGonzalez04 — CHANGES_REQUESTED** [review]
Wanted to see if you needed to pull in new PR's that were merged and check to make sure it was still functioning on your end.

File: components/Project/Project.tsx [hunk 2]
```diff
@@ -20,12 +20,6 @@ const Project = (props) => {
   const router = useRouter();
     return (
         <>
-            <HeaderNav
-              whichPage="profile"
```
[the remark this answers is not in the document]
Comment (NickGonzalez04) [reply components/Project/Project.tsx:24]: Never mind, I see in the `create-werk.tsx` where this is set for the HeaderNav. Disregard!

**Review by NickGonzalez04 — CHANGES_REQUESTED** [review]
Other than the button sizing in the`MarketPlace` page this is good to go!! Love it!

File: components/Marketplace/SortBounty.tsx [hunk 3]
```diff
@@ -44,18 +52,27 @@ export const SortBounty = ({ contracts, onChange }) => {
...
       >
         {contracts?.length || 0} results
       </Text>
+
+      <Button
+              alignSelf="flex-end" 
+              marginRight="84px !important"
+                variant={"primary"}
+                size={"lg"}
+                width={"309px"}
```
[the remark this answers is not in the document]
Comment (nagmak) [reply components/Marketplace/SortBounty.tsx:70]: LOL i see it now, aight i shall fix it ty for catching that

**Comment by nagmak** [comment]
Screen Shot 2022-05-31 at 8 05 12 PM

Made the change, looks like it's fixed on my end now @NickGonzalez04

**Comment by NickGonzalez04** [comment]
🔥🔥 Will do a once through again and approve but awesome!

**Review by NickGonzalez04 — APPROVED** [review]
Approval is good!

**Pull request merged**